cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml

```diff
@@ -1,9 +1,6 @@
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
-  labels:
-    prometheus: rook-prometheus
-    role: alert-rules
```

Comment:
Didn't transfer this label by accident, but actually not sure if this is useful to have by default